files/kubelet-config.json

```diff
@@ -25,9 +25,12 @@
   "hairpinMode": "hairpin-veth",
   "cgroupDriver": "cgroupfs",
   "cgroupRoot": "/",
+  "eventRecordQPS": 0,
   "featureGates": {
     "RotateKubeletServerCertificate": true
   },
+  "protectKernelDefaults": true,
+  "readOnlyPort": 0,
```

Comment:
Thanks!

Reply:
Is setting eventRecordQPS to 0 a good idea? from what I understand this will disable any rate limiting of event creation, are there any negatives to this?

Reply:
@stefansedich You are right that `eventRecordQPS` (`--event-qps`) defaults to 5, and that setting it to 0 will disable rate limiting. The article you linked to says that the default would be good for around 30 pods on a node.  

https://github.com/kubernetes/kubelet/blob/b81ee8851698006169372c9dce21feabc4bc3c99/config/v1beta1/types.go#L232-L239

That said, I have seen [several](https://www.stackrox.com/post/2019/09/12-kubernetes-configuration-best-practices/) [articles](https://rancher.com/docs/rancher/v2.x/en/security/hardening-2.2/) saying 0 is a good default. Do you think having 50 be a better default limit?

Reply:
@mogren I think the issue here is that the CIS benchmarks state to use `0` and I have never really seen the rationale behind this, to me it appears as though this param should be tuned on a per-cluster basis, I would love input from someone with more knowledge on the subject.

Reply:
@stefansedich Look at the source code.

`eventRecordQPS=0` is unlimited, which the CIS Benchmark warns can result in DOS.

Note that a value of 0 for `--event-qps` flag (deprecated) resulted in a value of 5, which was the default when this flag was omitted.

The default for `eventRecordQPS` (field) is also 5, but an explicit 0 is unlimited.

In other words: Do not add an explicit 0. Just omit it and it will default to 5.